crates/bevy_core/src/label.rs

```diff
@@ -13,7 +14,7 @@ use std::{
 };
 
 /// A collection of labels
-#[derive(Default, Reflect)]
+#[derive(Component, Default, Reflect)]
 #[reflect(Component)]
```

Comment:
Is there any chance we can wrap this into the derive, probably in a follow-up PR? That would make working with reflection much less frustrating.

Reply:
Yes, I believe this is one of the first things we might add. But I intentionally kept it as is to limit the scope of changes here. Details around automatic component reflection might need further discussion.